server/portal/apps/users/views.py

```diff
@@ -29,6 +29,19 @@ class AuthenticatedView(BaseApiView):
     def get(self, request):
         if request.user.is_authenticated:
             u = request.user
+            portal_roles = settings.PORTAL_ELEVATED_ROLES
```

Comment:
I think we'll want to move this main logic to the `launch_setup_checks` function, as we hit that when the user logs in. This view isn't necessarily used upon login